packages/palette_generator/example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +0,0 @@
-#Tue Jan 07 08:46:39 PST 2020
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
I thought this file was how we specified the gradle version to use; don't we have this in every Android example?

Reply:
It does if there is a gradlew file and we call gradlew. I cant find the tooling that uses the locally defined gradle. When I run `gradle tasks` from `packages/packages/palette_genrator/example/android` I get the some warnings about migration to gradle 8 but tasks show correctly. My hope/expectation was that I would see unit tests or something fail in ci so I could fix/migrate this instead. 

Let me spend some time figuring out all the ways this code could be tested and make sure it passes locally.

Reply:
`flutter build` creates the `gradlew` file. I believe `flutter build` and `flutter run` use it, and we also use it directly in the repo tooling here when running Android native unit tests. (That's why `config-only` will be useful; currently we have to make sure we run the unit tests after the test that everything builds)